data/remnant/remnant logistics.txt

```diff
@@ -0,0 +1,295 @@
+			`	"So, why did you offer this work to me?" you sing, trying to mimic the gestures you've seen that you think refer to one's self.`
+			`	He smiles and sings, "Very good! Very good! We schedule everything but we maintain slack in the system to take advantage of opportunities like the reactors being ready early, or to deal with emergencies. Sometimes, of course, we have to drop everything and sort the schedule out later. But, in normal times, we publish these jobs on the job board and captains that have capacity available can pick them up. The projects provide resource allocations to cover operating and crew costs, but there is no...what is that word? Margin! Yes! So the payment might not be what you are used to. Since your ship isn't in our planned resources, you provide extra capacity and we might be able to move some of our larger projects forward when you are in our space.`
+			choice
+				`	"Won't your pilots be upset at the competition?"`
+					goto competition
+				`	"Thanks for the opportunity."`
+					goto thanks
+			label competition
+			`	Eon smiles at the question. "You should talk to the other pilots and see. But, I think the ancestral space approach is different. We work more like how I imagine the navy does in the Republic. The ships are owned by the Remnant, but each captain is responsible for the ship, the crew, and the transport work that needs to be done. I think they will be happy to see new homes being built for families, and, "he winks at you, "spending more time with their own."`
+				goto thanks
+			label thanks
+			`	Before you can ask more about transport jobs, Eon takes out his commlink and taps on it for a few minutes. "I've added you to the job board. You should be able to see our current job list for each system when you check."`
+			`	You chat with Eon about how you've outfitted the <ship> as you finish your...coffee. Then, his commlink chirps. As he stands, he signs what you think means 'Goodbye' as he sings that he has to leave for another meeting. He walks off quickly with his head down looking at the datapad. Your commlink notifies you that <payment> has been added to your account.`
+
+trade
+	commodity "remnant civil"
+		"power distribution controls"
+		"sewage treatment modules"
+		"road grading and surfacing equipment"
+		"modular crane components"
+		"high tensile cables"
+		"earth moving equipment"
+		"heavy machinery tools"
+		"high voltage electrical diagnostic equipment"
+		"environmental heating and cooling systems"
+		"high volume pumps"
+		"environmental monitoring and reporting modules"
+		"construction robots"
+		"construction matrix mixers and extruders"
+		"worker housing modules"
+		"drilling equipment"
+		"waste recycling equipment"
+		"civil defense emergency supplies"
+		"portable machine shop modules"
+		"local network repeaters and uplinks"
+
+mission "Remnant: Logistics Civil Engineering"
```

Comment:
Making a few suggestions on this mission. If you wouldn't mind, please implement the changes I'm making here to the other missions below (Industrial, Outfitter, and Delivery) as well. Just don't want to clutter up or waste time repeating the same comment 3 more times for each mission.